### ISSUE TITLE
No gradient clipping in AdamW

### ISSUE THREAD

**Issue by OlegPlatonov** [huggingface/transformers#887]
Hi!

After moving from pretrained-bert to transformers I've noticed that the new AdamW optimizer does not perform gradient clipping, even though both BertAdam and OpenAIAdam used to do it. 

Also, in finetune_on_pregenerated example bias correction is turned off only for FusedAdam, but not for AdamW.

**Comment by thomwolf** [comment]
Yes, the LM fine-tuning example will be refactored.

Adding the removal of gradient clipping to the list of breaking changes, thanks.
